src/tcgwars/logic/impl/gen8/ChillingReign.groovy

```diff
@@ -2218,7 +2218,7 @@ public enum ChillingReign implements LogicCardInfo {
             delayedA {
               after PROCESS_ATTACK_EFFECTS, {
                 bg.dm().each {
-                  if (it.from == self && it.to.benched && (it.to.pokemonV || it.to.pokemonVMAX || it.to.pokemonGX) && it.dmg.value) {
+                  if (it.from.owner == self.owner && it.from.rapidStrike && it.to.benched && (it.to.pokemonV || it.to.pokemonVMAX || it.to.pokemonGX) && it.dmg.value) {
```

Comment:
I don't see anything here for
> You can't use more than 1 Throwing Coach Ability at a time.

Also PokemonVMAX are PokemonV so no need for both checks, only PokemonV is needed.

```suggestion
                  if (it.from.owner == self.owner && it.from.rapidStrike && it.to.benched && (it.to.pokemonV || it.to.pokemonGX) && it.dmg.value) {
```

Reply:
Marked unresolved. Only the second half was handled.

Reply:
Thanks for catching that, I've added it in my latest commit